### PR TITLE
Add new benchmarking scripts and results

### PR DESCRIPTION
**Pull request by ofuhrer**
This PR adds the new c96, c128, c192, c256 benchmarks with Intel and GNU compiler on Piz Daint.

### CODE REVIEW

File: benchmarking/daint_single_node/run_different_resolutions.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+workdir=/scratch/snx3000/olifu/benchmarking
+\rm -rf "${workdir}"
+partition="normal --no-wait"
+#partition="debug"
```
Comment (twicki) [inline benchmarking/daint_single_node/run_different_resolutions.sh:6]: can we remove the comment?

File: benchmarking/daint_single_node/run_different_resolutions.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+for config in c96 c128 c192 c256 ; do
+for compiler in gnu intel ; do
```
Comment (twicki) [inline benchmarking/daint_single_node/run_different_resolutions.sh:9]: can we indent the for loops?

**Review by twicki — CHANGES_REQUESTED** [review]

File: benchmarking/daint_single_node/run_different_resolutions.sh (hunk 1 above)
Comment (ofuhrer) [reply benchmarking/daint_single_node/run_different_resolutions.sh]: done

File: benchmarking/daint_single_node/run_different_resolutions.sh (hunk 2 above)
Comment (ofuhrer) [reply benchmarking/daint_single_node/run_different_resolutions.sh]: done

**Review by twicki — APPROVED** [review]
lgtm

**Pull request merged**